### ISSUE TITLE
Error thrown in debounce stream never ends

### ISSUE THREAD

**Issue by kdeloach** [baconjs/bacon.js#560]
When an error is thrown inside a debounced event stream, the error is continuously emitted.

This error occurs on the latest release but not in earlier releases. I narrowed down the breaking commit to 4794940.

Here's the code that reproduces the issue:

    Bacon.once(1).debounce(1000).onValue(function() {
        throw 'Error';
    });

In release 0.7.23 the output is:

> Uncaught Error

In release 0.7.50 the output is:

> Uncaught Error
> Uncaught Error
> Uncaught Error
> Uncaught Error
> Uncaught Error
> ...

PR with a fix will be submitted shortly.

**Comment by raimohanska** [comment]
Cannot reproduce this. For me, output with node 0.10.32 is this:

```
/Users/juha/code/bacon.js/testing/560.js:4
      throw 'Error';
      ^
Error
```

Can you bake a runnable example in jsbin/jsfiddle etc?

Regarding the fix, there always need to be a test case that fails without the new fix.

**Comment by raimohanska** [comment]
Oh well, in a browser I can indeed reproduce this.

**Comment by raimohanska** [comment]
There's a fix in master: 80661af9e9c1c1732e19c39f1873c399e4de7a94

A test case would be nice too.

**Comment by raimohanska** [comment]
Struggling with writing a test case :(

**Comment by kdeloach** [comment]
I should have mentioned that this is browser related. Next time I will include a jsfiddle.
I could take a stab at providing a test case, but my knowledge of Bacon and CoffeeScript is woefully lacking. It might take me a while.

**Comment by raimohanska** [comment]
I'll give the testcase another try later tonight, hopefully. Interesting challenge :) 

Bacon.js doesn't really handle exceptions at all, all it tries to do is to stay in a consistent state after one occurs.

**Comment by raimohanska** [comment]
Ok, the test is in.

**Comment by raimohanska** [comment]
Fix included in 0.7.52 which will be out any minute.

**Comment by raimohanska** [comment]
And when I say 0.7.52, I naturally mean 0.7.53.
